### ISSUE TITLE
Add 'learning curve' function to 'test and score'?

### ISSUE THREAD

**Issue by gengyabc** [biolab/orange3#5980]
**What's your use case?**
I wish I can see the learning curve of my model, so I know if my model is overfitted.

**What's your proposed solution?**
I think the 'test and score' widget could add this function. More specifically, maybe merge  'test and score' and 'learningCurveC' in the demo.

Thank you guys

**Comment by BlazZupan** [comment]
You are probably referring to the learning curve where we subsample the training set and observe the accuracy on the test set. This was implemented in learningCurveC, and if placed in Orange, should require a new widget. Please confirm if this is the case. If, may introduce such widget if some foreseable future.

**Comment by zy-tech** [comment]
Yes, this is it. 
But I wish if the 'loss VS epoch' can be plotted, including training and testing loss.

**Comment by gengyabc** [comment]
Hi, I found that learningCurveC will give us a list of value, and I thought it was 'some value' vs epoch. But as you said, do you mean it was 'sample of each group' vs 'group'?
What I wish is that there is a 'loss vs epch' curve, like in [TensorFlow playground](https://playground.tensorflow.org/) as shown below circled in red
![image](https://user-images.githubusercontent.com/12781437/172029193-50f8a600-840f-438f-b503-c2b1ddb77753.png)

**Comment by lanzagar** [comment]
As mentioned by @BlazZupan, learningCurveC does something else - evaluates the accuracy at different training set sizes, and not for different training lengths (epochs/iterations).
As for your wish, it is then model dependent, and would not belong in the Test and Score as much as in a specific widget (for example in the Neural Network widget). We had a discussion and agree it would be a nice feature to have, but currently nobody found it pressing enough to allocate time and resources for it at this time.
